src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -10,6 +10,11 @@ public class Manufacturer {
     public Manufacturer() {
     }
 
+    public Manufacturer(String name, String country) {
+        this.name = name;
+        this.country = country;
+    }
+
```

Comment:
remove it, and replace calling this constructor with `Manufacturer(null, String name, String country)`